survey-page/add-to-results.js

```diff
@@ -0,0 +1,23 @@
+import { checkArray } from './check-results.js';
+
+export const addToResult = (object) => {
```

Comment:
FYI, you don't need the parens if your arrow function has a single parameter. You can use ... = object => 
There's no problem with using them though!